apps/crn-server/src/data-providers/contentful/reminder.data-provider.ts

```diff
@@ -165,8 +165,14 @@ export class ReminderContentfulDataProvider implements ReminderDataProvider {
       timezone,
     );
 
+    const versionReminderIds = publishedResearchOutputVersionReminders.map(
+      (i) => i.data.researchOutputId,
```

Comment:
```suggestion
      (reminder) => reminder.data.researchOutputId,
```

`i` would suggest an iterator which this isn't one